streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -185,6 +185,10 @@
     public static final String APPLICATION_SERVER_CONFIG = "application.server";
     private static final String APPLICATION_SERVER_DOC = "A host:port pair pointing to an embedded user defined endpoint that can be used for discovering the locations of state stores within a single KafkaStreams application";
 
+    /**{@code rack.id} */
+    public static final String RACK_ID_CONFIG = "rack.id";
+    private static final String RACK_ID_DOC = "An identifier for the rack of a node for rack-aware replication.";
```

Comment:
"replication" might be confusion in this context. What about "StandbyTask assignment"

nit: Can you add this config in alphabetical order. Thx.